app/app.tsx

```diff
@@ -35,29 +37,33 @@ EStyleSheet.build({
   // $textColor: '#0275d8'
 })
 
+// FIXME should we only load the currently used local?
+// this would help to make the app load faster
+// this will become more important when we add more languages
+// and when the earn section will be added
+//
+// alternatively, could try loadAllLocalesAsync()
```

Comment:
On my local file system each locale takes up ~60kb.  I think we could load it async without taking up too much user bandwidth.